completion/_catkin

```diff
@@ -500,11 +503,13 @@ case "$state" in
           _describe -t terminators "terminate multiword argument" terminating_opts
         elif ((${words[1,$CURRENT-1][(I)--make-args]} > ${words[1,$CURRENT-1][(I)--]} && ${words[1,$CURRENT-1][(I)--make-args]} >= ${words[1,$CURRENT-1][(I)--*make-args]} && ${words[1,$CURRENT-1][(I)--make-args]} > ${words[1,$CURRENT-1][(I)--(black|white)list]})); then
           # same as above, just for make
+          _debug_log "make"
           _dispatch $service make
           _describe -t terminators "terminate multiword argument" terminating_opts
         elif ((${words[1,$CURRENT-1][(I)--catkin-make-args]} > ${words[1,$CURRENT-1][(I)--]} && ${words[1,$CURRENT-1][(I)--catkin-make-args]} >= ${words[1,$CURRENT-1][(I)--*make-args]} && ${words[1,$CURRENT-1][(I)--catkin-make-args]} > ${words[1,$CURRENT-1][(I)--(black|white)list]})); then
-          # same as above, just for catkin-make
-          # TODO implement
+          # same as above, just for catkin_make
+          _debug_log "catkin_make"
+          _dispatch $service catkin_make
```

Comment:
Is there already completion for `catkin_make`? At least for me, this does not complete anything, but if it works for you it might be an issue with my setup.

Reply:
Unfortunately, completion is still only for bash (https://github.com/ros/catkin/tree/noetic-devel/cmake/env-hooks). However, by allowing a search for it, the functionality doesn't change. So we can use it, and later when zsh completion is available, this completion doesn't have to be updated.

Reply:
I agree, it is better than doing nothing and this would not be the place to implement zsh completion for `catkin_make`.